### DISCUSSION TITLE
VirtualTable 表格按照文档无法使用 参照版本 [4.24.12] 😭

### DISCUSSION THREAD

**Discussion by wuxinheng** [ant-design/ant-design#51615] [Q&A] [answered]
我的具体问题代码以及提示：
![image](https://github.com/user-attachments/assets/6f8bbe8a-cdc0-4d61-b2f6-2de050dc279d)

我参考的文档路径：
https://4x-ant-design.antgroup.com/components/table-cn/?locale=zh-CN#components-table-demo-virtual-list

我参考的代码：
https://paste.ofcode.org/XwECE2JZKZERtNFdvdSC6f

我的 package dependencies信息：

-  "@ant-design/icons": "^4.6.3",
-  "@ant-design/pro-card": "1.14.14",
-  "@ant-design/pro-descriptions": "1.9.7",
-  "@ant-design/pro-field": "1.20.0",
-  "@ant-design/pro-form": "1.36.0",
-  "@ant-design/pro-layout": "6.24.2",
-  "@ant-design/pro-list": "1.12.2",
-  "@ant-design/pro-table": "2.50.0",
-  "@ant-design/pro-utils": "1.22.0",
-  "@microsoft/signalr": "^3.1.0",
-  "antd": "4.17.4",
-  "antd-img-crop": "3.14.1",
-  "classnames": "^2.2.6",
-  "famfamfam-flags": "^1.0.0",
-  "lodash": "^4.17.11",
-  "moment": "^2.24.0",
-  "oidc-client": "^1.10.1",
-  "omit.js": "^1.0.2",
-  "path-to-regexp": "2.4.0",
-  "qs": "^6.9.0",
-  "react": "^16.13.1",
-  "react-cookie": "^4.0.1",
-  "react-dom": "^16.8.6",
-  "react-helmet-async": "^1.0.4",
-  "react-window": "1.8.5",
-  "umi": "^3.0.14",
-  "umi-request": "^1.0.8",
-  "use-merge-value": "^1.0.1"

**Comment by petercat-assistant** [comment]
看起来您在使用 Ant Design 的 VirtualTable 时遇到了问题。根据您提供的信息，您参考的文档版本是 4.24.12，而您当前使用的 `antd` 版本是 4.17.4。这可能是导致问题的原因之一，因为您的 `antd` 版本可能不支持文档中提到的某些功能或 API。

以下是一些建议来帮助您解决问题：

1. **升级 Ant Design 版本**: 考虑将 `antd` 升级到与文档版本一致的 4.24.12 或更高版本。这可以确保您使用的功能与文档中的描述一致。

2. **检查依赖关系**: 确保所有相关依赖项（如 `react` 和 `react-dom`）的版本与 `antd` 兼容。

3. **查看错误信息**: 仔细查看控制台中的错误信息，可能会提供有关问题的更多线索。

4. **参考示例代码**: 确保您的代码与文档中的示例代码一致，特别是在配置和使用 VirtualTable 时。

如果您已经尝试了这些步骤但问题仍然存在，请提供更多详细信息或错误信息，以便我能更好地帮助您。

如需进一步帮助，请在评论中描述您的问题，并@petercat-assistant与我开始对话。

**Reply by wuxinheng** [reply]
我无法升级版本

**Comment by wuxinheng** [comment]
已经看了相关的更新日志，其实我很想找到4.17.4 版本的文档，但是我没有找到😕

**Reply by zombieJ** [reply]
可以 checkout repo 到 4.17.4 的 Tag 然后 npm start 就是文档了

**Reply by wuxinheng** [reply]
谢谢，AI 已经解决了

**Reply by wuxinheng** [reply]
示例代码

https://paste.ofcode.org/37XQFwBgFsw6sqcFxnVAptn

**Comment by wuxinheng** [comment, marked as answer]
https://paste.ofcode.org/37XQFwBgFsw6sqcFxnVAptn
